demo/model.py

```diff
@@ -0,0 +1,60 @@
+# Import dependencies
+from pathlib import Path
+import joblib
+import os
+import numpy as np
+import h5py
+import subprocess
+
+BASE_DIR = Path(__file__).resolve(strict=True).parent
+
+# Create fasta file
+def create_temp_fa(embedding):
+    path_fa = Path(BASE_DIR).joinpath("temp_fa.fa")
+    Seq = {}
+    Seq['header'] = embedding
+    with open(path_fa, 'w') as fout:
+        for header, sequence in Seq.items():
+            fout.write(f">{header}\n{sequence}\n")
+    return path_fa
+
+# Get model details
+def model_details(task):
+    models_dir = Path(BASE_DIR.parent).joinpath('saved_models/best_models')
+    for file in os.listdir(models_dir):
+        if file.startswith(task):
+            model_name  = file
+
+    model_file = Path(BASE_DIR).joinpath(models_dir, model_name)
+    pool = model_name.split('_')[2]
+    pt_model =  f"prose_{model_name.split('_')[1]}"
+    return pt_model, pool, model_file
+
+# Calculate embeddings
```

Comment:
Not really a big deal, but I would probably change this to "Compute Embeddings" instead of calculate.